charts/rollup/Chart.yaml

```diff
@@ -15,14 +15,19 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.4
+version: 0.5.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "0.4.0"
 
+dependencies:
```

Comment:
We'll have to see if we hit an issue here or not in argo land. Argo can be weird around helm update dependency if its in chart.yml vs requirements.yml but I dont recall the exact

Reply:
I have been able to test this and it seems to work fine in argocd.